src/grandorgue/config/GOConfig.cpp

```diff
@@ -215,6 +215,8 @@ GOConfig::GOConfig(wxString instance)
     MidiRecorderPath(
       this, wxT("General"), wxT("MIDIRecorderPath"), wxEmptyString),
     MidiPlayerPath(this, wxT("General"), wxT("MIDIPlayerPath"), wxEmptyString),
+    CheckForUpdatesAtStartup(
```

Comment:
@rousseldenis @larspalo do you agree with enabling checking for updates by default? If most of organs do not have an internet connnection, is it better set to false?

Reply:
@oleg68 Perhaps, but I'd rather not have any notification at all if the checking would fail for any reason.

Reply:
Yes, agree with @larspalo too on notifications.